docs/source/api/user_login_oauth.rst

```diff
@@ -0,0 +1,71 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-user-login-oauth:
+
+********************
+``user/login/oauth``
+********************
+
```

Comment:
You should use a `.. versionadded::` directive to clarify in what API version(s) this endpoint is available.

Reply:
done